content/sensu-enterprise/2.6/guides/using-contact-routing.md

```diff
@@ -0,0 +1,623 @@
+metric integrations.
+
+_NOTE: Sensu Enterprise supports contact routing for all integrations except
+EC2, Puppet, Chef, Flapjack, and Event Stream._
+
+In this guide we'll cover configuring and using Sensu Enterprise Contact
+
+## Contact Routing Basics
+
+In Sensu Enterprise Contact Routing, contacts are composed of a name and
+configuration overrides for one or more of Sensu Enterprise's built-in
+integrations or handler configurations. A contact in Sensu Enterprise is not too
+dissimilar from a contact on your phone, which usually have a name and one or
+more _identifiers_ for various communication **channels** (e.g. a **phone**
+_number_, **email** _address_, **Twitter** _username_, etc).
+
+### Contact Configuration
+
+Configuring a contact requires you to define the name of the contact and the
+services plus configuration override for that contact.
+
+{{< highlight json >}}
+{
+  "contacts": {
+    "support": {
+      "email": {
+        "to": "support@sensuapp.com"
+      },
+      "slack": {
+        "channel": "#support"
+      }
+    }
+  }
+}
+{{< /highlight >}}
+
+In the example above we have a contact named **support**. For **support** we
+have an override configuration for **email** and **slack**. In our example we're having
+this contact email _support@sensuapp.com_ instead of the email integration's
+default **to** address. For Slack, we're defining the channel the event should
+post to, in this case _#support_.
+
+### Check Configuration
+
+Once a contact has been defined, we can now apply which contact(s) to use in a
+check configuration. The use case for having it on a check could be that the
+client is owned by a particular team or group, the same can be done for specific
+checks. For instance:
+
+{{< highlight json >}}
+{
+  "checks": {
+    "example_check": {
+      "command": "do_something.rb",
+      "handlers": [
+        "email"
+      ],
+      "contacts": [
+        "support"
+      ]
+    }
+  }
+}
+{{< /highlight >}}
+
+### Client Configuration
+
+A client configuration can include a contact to use. This is helpful for defining ownership of the client itself.
+
+{{< highlight json >}}
+{
+  "client": {
+    "name": "ec2-west-id-12345",
+    "address": "8.8.8.8",
+    "subscriptions": [
+      "production",
+      "aws"
+    ],
+    "contact": "aws-team"
+  }
+}
+{{< /highlight >}}
+
+# Example Implementations
+
+In this section we will be showcasing how contact routing interacts depending on the different possible configurations.
+
+## Global Configuration
+
+For the examples we'll be working with we want to define what our global handlers will be.
+
+In the following section we'll be going over different scenarios and showcasing how contact routing would be used and how contact routing affects notifications being sent. We'll be using Email, Slack and PagerDuty as our example integrations/handlers.
+
+Bellow is the default configuration for the integrations we'll be working with in our examples:
```

Comment:
```suggestion
Below is the default configuration for the integrations we'll be working with in our examples:
```

Reply:
I'd stress more that these handlers are used with all examples. Maybe suggest to copy them to a text editor to follow a long with.